Sources/Bugsnag/PayloadTransformer.swift

```diff
@@ -3,17 +3,25 @@ import Stacked
 import HTTP
 
 public protocol PayloadTransformerType {
-    func payloadFor(message: String, metadata: Node?, request: Request?) throws -> JSON
+    func payloadFor(message: String, metadata: Node?, request: Request?, filters: [String]) throws -> JSON
```

Comment:
I've been thinking about where the filters should come from.. maybe we should use the filters from the function inside of the payloadtransformer, and then use the configuration filters on the reporter? This way, we can add filtering per report call later on we need to.